CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Below are suggested reviewers for PRs to packages in this repository.
+#
+# These are suggestions; it is fine to have your changes reviewed by someone
+# else.
+
+/pkgs/checks/          @dart-lang/core-package-admins
```

Comment:
cc @natebosch and @jakemac53 re: whether these 5 packages match up w/ what you want auto-assigned for review

Reply:
LGTM